setup/createKubernetesServiceAccounts.sh

```diff
@@ -57,7 +56,7 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: dynatrace-role
+  name: teleport-role
```

Comment:
replace it with 'dynatrace-role'?

Reply:
I agree - just updated and pushed the change